src/components/DuosDatePicker.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import {createTheme, ThemeProvider} from '@mui/material/styles';
+import {
+  DesktopDatePicker,
+  LocalizationProvider,
+  PickersActionBarProps,
+  PickersDay,
+  PickersDayProps
+} from '@mui/x-date-pickers';
+import {AdapterDayjs} from '@mui/x-date-pickers/AdapterDayjs';
+import {Button} from '@mui/material';
+import DialogActions from '@mui/material/DialogActions';
+import {Dayjs} from 'dayjs';
+import type {} from '@mui/x-date-pickers/themeAugmentation';
+
+
+interface DUOSDatePickerProps {
+    inputFormat: string;
+    value: Dayjs;
+    onChange: Function;
+    onError: Function;
+    readOnly: boolean;
+}
+
+export const DuosDatePicker = (props: DUOSDatePickerProps) => {
+  const {inputFormat, value, onChange, onError, readOnly} = props;
+  const theme = createTheme({
+    palette: {
+      primary:{
+        main:'#216FB4',
+      },
+      secondary:{
+        main:'#ffffff',
+        contrastText:'#216FB4'
+      },
+    },
+    components: {
+      MuiButton:{
+        styleOverrides:{
+          text:{
+            fontFamily: 'Montserrat',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px 7px 20px',
+          },
+        },
+      },
+      MuiIconButton: {
+        styleOverrides: {
+          sizeMedium: {
+            color:'#216FB4',
+          },
+        },
+      },
+      MuiDayCalendar:{
+        styleOverrides:{
+          weekDayLabel:{
+            fontFamily: 'Montserrat',
+            color: '#000',
```

Comment:
How would you feel about standardizing this on `black`/using the color names where we have them?

Reply:
I don't have strong feelings about this and would be happy to make that change, however we should be on the same page about colors in the context of MUI.  Do you expect to see [MUI color palettes](https://mui.com/material-ui/customization/color/#color-palette) or CSS ? For this PR, do you expect that standardization across all of DUOS-UI or just in the context of the DuosDatePicker?

Reply:
One minor thing I've observed in just using the CSS color 'black' instead of '#000' is that the IDE removes the option for the color picker in the gutter.  Does that matter to you?

Reply:
For black I don't strongly care - if we need to change it we can always move it to get the color picker back.

I would personally say let's steer towards unity across the codebase, so if the rest of the codebase uses `#000` then lets do that. However, I think right now there's existing inconsistency?